main.js

```diff
@@ -65,6 +66,44 @@ function getCaptureError (client, _captureError) {
 	};
 }
 
+function sanitiseError (err) {
+	//Initially check for a specific email field e.g. in a url in an attempt to not blitz helpful error information if possible
+	if (err.message) {
+		const origErrorMsg = err.message
+		const emailFieldRegex = 'email='
+		const indxEmailField = origErrorMsg.search(emailFieldRegex)
+
+		let cleanedErr = origErrorMsg
+		if (indxEmailField > 0) {
+
+			cleanedErr = origErrorMsg.slice(0,indxEmailField) + 'email=-redacted-'
+
+			//in case there are more fields / is further at the end of the string
+			let indexOfAdditionalParams = origErrorMsg.indexOf('&', indxEmailField)
+			let indexOfPostURLspace = origErrorMsg.indexOf(' ', indxEmailField)
+
+			if(indexOfAdditionalParams > 0) {
+				cleanedErr = cleanedErr + origErrorMsg.slice(indexOfAdditionalParams, err.length)
+			}
+			else if (indexOfPostURLspace > 0){
+				cleanedErr = cleanedErr + origErrorMsg.slice(indexOfPostURLspace, err.length)
+			}
+
+		}
+
+		//The catch all check in case there are any remaining email addresses in the error
+		const emailRegex = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/g;
```

Comment:
Oh dear, good point, it was a copy-paste during development that I meant to look into further before committing, originally from here:
http://www.regular-expressions.info/email.html

Partway down the page, at this section:

> We get a more practical implementation of RFC 5322 if we omit IP addresses, domain-specific addresses, the syntax using double quotes and square brackets. It will still match 99.99% of all email addresses in actual use today.
> 
> \A[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@
> (?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\z

Perhaps there is a better, preferred alternative?